README.md

```diff
@@ -1,70 +1,20 @@
 # Time Deposit Refactoring Kata
-
 ## XA bank time deposit
 
-It's unfortunate that the junior developer has left the company without handing his work over to anyone. And what is even more unfortunate is that you have been assigned to take over the code that he left behind.
+It's unfortunate that the junior developer has left the company without handing his work over to anyone. And what is even more unfortunate is that you have been assigned to take over the code that he left behind. 
 
 We were able to savage only a portion of the business requirements, but we're not sure if the existing implementation has covered all the flows.
```

Comment:
salvage?

Reply:
there's just 1 extra space, the paragraph is still there

Reply:
Sorry, I meant to say 'salvage' instead of 'savage'.

Reply:
fixed